Library/Homebrew/cleanup.rb

```diff
@@ -196,19 +205,20 @@ def periodic_clean_due?
       PERIODIC_CLEAN_FILE.mtime < CLEANUP_DEFAULT_DAYS.days.ago
     end
 
-    def periodic_clean!
-      return false unless periodic_clean_due?
+    def self.periodic_clean!(dry_run: false)
+      return if Homebrew::EnvConfig.no_install_cleanup?
+      return unless periodic_clean_due?
 
-      if dry_run?
-        ohai "Would run `brew cleanup` which has not been run in the last #{CLEANUP_DEFAULT_DAYS} days"
+      if dry_run
+        oh1 "Would run `brew cleanup` which has not been run in the last #{CLEANUP_DEFAULT_DAYS} days"
       else
-        ohai "`brew cleanup` has not been run in the last #{CLEANUP_DEFAULT_DAYS} days, running now..."
+        oh1 "`brew cleanup` has not been run in the last #{CLEANUP_DEFAULT_DAYS} days, running now..."
```

Comment:
Why has this been changed from `ohai` to `oh1`? It's not that important to me either way; I'm just curious.

Reply:
`periodic_clean!` used to be invoked in place of `install_formula_clean!` but now that it’s invoked on its own, `oh1` makes more sense.

Reply:
I see. That makes sense.